etc/kernelspecs/dask_python_yarn_remote/bin/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [ "${EG_IMPERSONATION_ENABLED}" = "True" ]; then
+    IMPERSONATION_OPTS="--user ${KERNEL_USERNAME:-UNSPECIFIED}"
+    USER_CLAUSE="as user ${KERNEL_USERNAME:-UNSPECIFIED}"
+else
+    IMPERSONATION_OPTS=""
+    USER_CLAUSE="on behalf of user ${KERNEL_USERNAME:-UNSPECIFIED}"
+fi
+
+echo
+echo "Starting IPython kernel for Dask ${USER_CLAUSE}"
+echo
+
+PROG_HOME="$(cd "`dirname "$0"`"/..; pwd)"
+
+set -x
+eval exec \
+     "${DASK_YARN_EXE}" submit \
+     "${DASK_OPTS}" \
```

Comment:
Minimal, but EXE seems like Windows, how about _SCRIPT ?

Reply:
It's not really a script. Perhaps `DASK_YARN_CLI` or `DASK_YARN`?

Reply:
Let's leave like it is for now